helpers/helpers.html

```diff
@@ -0,0 +1,44 @@
+<!--
+@license
+Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<link rel="import" href="../../polymer/polymer.html">
+
+<script>
+  Polymer.AppLayout = Polymer.AppLayout || {};
+
+  Polymer.AppLayout.scrollEffects = Polymer.AppLayout.scrollEffects || {};
+
+  /**
+   * Helper function that scrolls to a particular
+   * set of coordinates in a scroll target without scroll effects transitions.
+   *
+   * @param {number} scrollLeft
+   * @param {number} scrollTop
+   * @param {HTMLElement?} scrollTarget
+   */
+  Polymer.AppLayout.scrollTo = function scrollTo(scrollLeft, scrollTop, scrollTarget) {
```

Comment:
I am seeing this helper function is useful because I can just call this to scroll to some position. Without this, I need to know for window (document scroll) I need to use `window.scrollTo(x,y)` and others I need to use `elt.scrollLeft/Right = x/y`.  The silent part could be just an add-on.